frontend/src/styles/element-dark.scss

```diff
@@ -460,4 +460,14 @@ html.dark {
     .el-checkbox__input.is-indeterminate .el-checkbox__inner::before {
         background-color: var(--panel-main-bg-color-10);
     }
+
+    .custom-input-textarea {
+        background-color: var(--panel-main-bg-color-10) !important;
+        color: var(--el-color-info) !important;
+    }
+    .custom-input-textarea:hover {
+        background-color: var(--panel-main-bg-color-9) !important;
+        color: var(--el-color-primary) !important;
+    }
+
 }
```

Comment:
The code additions appear to be adding custom styles for input textareas that fit into a dark theme context of an element being styled with `.dark`. The changes include modifying:

- The background color and text color for regular state (`var(--panel-main-bg-color-10)` and `var(--el-color-info)`) 
- Hover states with increased contrast (`var(--panel-main-bg-color-9)` and `var(--el-color-primary)`) 

These improvements enhance readability on dark backgrounds, making them more user-friendly. However, it's worth noting that these customizations assume a specific CSS framework (like Element UI) where variables like `--panel-main-bg-color-10` and `--el-color-info` exist. If those variable names change or if there's another way to style these elements without altering the framework-specific ones, further adjustments would be necessary to ensure compatibility and maintainability.